sklearn/model_selection/_split.py

```diff
@@ -1893,6 +1897,9 @@ def check_cv(cv=3, y=None, classifier=False):
         The return value is a cross-validator which generates the train/test
         splits via the ``split`` method.
     """
+    warnings.warn("The default value of n_splits=3 is deprecated"
+                  " in version 0.20 and will be changed to "
+                  "n_splits=5 in version 0.22", DeprecationWarning)
     if cv is None:
```

Comment:
I guess the warning should only be displayed if cv is None, as it will take the default value in that case.

Reply:
Line 358 comment still has the following:
   

>   n_splits : int, default=3
>         Number of folds. Must be at least 2.

This comment should be updated as well.